astroquery/ibe/__init__.py

```diff
@@ -3,43 +3,14 @@
 IRSA Image Server program interface (IBE) Query Tool
 ====================================================
 
-This module contains various methods for querying the
-IRSA Image Server program interface (IBE).
+The ``astroquery.ibe`` module is deprecated and will be removed in a future
+release of Astroquery. IRSA no longer supports the IBE interface. To access
+IRSA datasets programatically, users encouraged to use the TAP protocol
+(http://irsa.ipac.caltech.edu/docs/program_interface/TAP.html) via PyVO
+(https://pyvo.readthedocs.io/en/latest/dal/index.html#table-access-protocol).
```

Comment:
I am pretty sure the Astropy Project wants to encourage users to use `astropy.utils.tap` and not redirect them to the `pyvo` package. The TAP functionality currently in astroquery works with the main Gaia DR1 archives in Europe but some small changes are needed to make it work with IRSA TAP.

Reply:
(oh sorry -- this was addressed in the main conversation already)